src/core/manager.c

```diff
@@ -410,6 +410,9 @@ static int manager_setup_signals(Manager *m) {
 
         assert(m);
 
+        if (m->test_run)
+                return 0;
```

Comment:
This matches what other functions do: it is easier to put the m->test_run test at the top of each function because than it's next to the stuff being skipped:
```
src/core/manager.c-static int manager_setup_notify(Manager *m) {
src/core/manager.c-        int r;
src/core/manager.c-
src/core/manager.c:        if (m->test_run)
src/core/manager.c-                return 0;
--
src/core/manager.c-static int manager_setup_kdbus(Manager *m) {
src/core/manager.c-        _cleanup_free_ char *p = NULL;
src/core/manager.c-
src/core/manager.c-        assert(m);
src/core/manager.c-
src/core/manager.c:        if (m->test_run || m->kdbus_fd >= 0)
src/core/manager.c-                return 0;
--
src/core/manager.c-static int manager_connect_bus(Manager *m, bool reexecuting) {
src/core/manager.c-        bool try_bus_connect;
src/core/manager.c-
src/core/manager.c-        assert(m);
src/core/manager.c-
src/core/manager.c:        if (m->test_run)
src/core/manager.c-                return 0;
--
src/core/manager.c-
src/core/manager.c-static void manager_notify_finished(Manager *m) {
src/core/manager.c-        char userspace[FORMAT_TIMESPAN_MAX], initrd[FORMAT_TIMESPAN_MAX], kernel[FORMAT_TIMESPAN_MAX], sum[FORMAT_TIMESPAN_MAX];
src/core/manager.c-        usec_t firmware_usec, loader_usec, kernel_usec, initrd_usec, userspace_usec, total_usec;
src/core/manager.c-
src/core/manager.c:        if (m->test_run)
src/core/manager.c-                return;
```
The only exception is `manager_setup_cgroup`, but that one is different because the part being skipped is a chunk in the middle of the funcition.